en/micro-integrator/docs/references/connectors/db-event-inbound-endpoint/db-event-inbound-endpoint-example.md

```diff
@@ -0,0 +1,139 @@
+
+Hence, for an enterprise integration platform it is a useful feature to be able to generate events based on the data changes. **DB Event Inbound Endpoint** is the DB event listener for `WSO2 Enterprise Integrator`. You can configure it with any popular Database systems such as `MySQL` and `Oracle` etc. 
+
+Following are the main features of the event generator. 
+
+1. Trigger an event with the data of table row when a new record is added or updated. Optionally, delete the row associated with the event after triggering the event
+2. Trigger an event when a boolean field is flipped in a particular table row. 
+
+## What you'll build
+
+In this example let us see how to configure `DB-event Inbound Endpoint` so that it can listen to data changes done to a `MySQL` table. Out of the features mentioned above feature no:1 is used here. Please refer to [reference guide](db-event-inbound-endpoint-config.md) if you need to use other features. 
+
+In the system of an enterprise a relational database table is used to store customer information. Customers' information is added by an external system to the database which is not in enterprise's control. As soon as a new customer is inserted, the system need to pick up and process its data. `WSO2 EI` is used here to listen to DB changes and invoke the relevant processes. It can invoke backend APIs or place data into a message bus after required data transformations. However, for simplicity of this example, we will just log the message. You can extend the sample as required using WSO2 EI mediators. 
+
+Following diagram shows the overall solution we are going to build.
+
+<img src="/assets/img/connectors/db-event-diagram.png" title="Overview of DB event inbound EP use case" width="600" alt="Overview of DB event inbound EP use case"/>
+
+
+## Setting up the environment 
+
+First, install [MySQL database](https://www.mysql.com/downloads/) locally. If you have a remote server, please obtain credentials required to connect. Here in this example database credentials are assumed as username=`root` and password=`root`. 
+
+1. Create a database called `test`. Then create a table called `CDC_CUSTOM` under that database using following SQL script. 
+```sql
+CREATE TABLE `test`.`CDC_CUSTOM` (
+  `ID` INT NOT NULL,
+  `NAME` VARCHAR(45) NULL,
+  `ADDRESS` VARCHAR(45) NULL,
+  `AMOUNT` INT NULL,
+  PRIMARY KEY (`ID`));
+```
+
+2. In order to track new records we need an additional column. If you apply this feature to an existing database table, you can alter the table as below. It will add a column of type `TIMESTAMP` which gets automatically updated upon insert or update of a record. 
```

Comment:
```suggestion
2. We need an additional column in order to track new records. If you apply this feature to an existing database table, you can alter the table as shown below. It will add a column of type `TIMESTAMP`, which gets automatically updated when you insert or update a record. 
```

Reply:
Updated.